src/scripts.js

```diff
@@ -30,17 +38,26 @@ const fetchData = () => {
 };
 
 const handleData = (data) => {
```

Comment:
possibly a little generic for the naming, but I know what you mean and run into this all the time too!